diff --git a/Resources/views/Default/index.html.twig b/Resources/views/Default/index.html.twig
index 5ebd2f6..b02a6a1 100644
--- a/Resources/views/Default/index.html.twig
+++ b/Resources/views/Default/index.html.twig
@@ -15,6 +15,8 @@
   {% if app.environment == 'prod' %}
     <script src="{{ asset('bundles/os2displayscreen/assets/build/assets.min.js') }}"></script>
     <script src="{{ asset('bundles/os2displayscreen/assets/build/app.min.js') }}"></script>
+    <script src="{{ asset('bundles/bibsdbsonderborgcalendar/assets/libs/angular-filter.min.js') }}"></script>
+    <script src="{{ asset('bundles/bibsdbsonderborgcalendar/apps/toolsModule/filters/groupedEvents.js') }}"></script>
   {% else %}
     <script src="{{ asset('bundles/os2displayscreen/assets/lib/jquery-1.11.1.min.js') }}"></script>
     <script src="{{ asset('bundles/os2displayscreen/assets/lib/angular.min.js') }}"></script>
@@ -22,6 +24,7 @@
     <script src="{{ asset('bundles/os2displayscreen/assets/lib/angular-css-injector-1.0.4.js') }}"></script>
     <script src="{{ asset('bundles/os2displayscreen/assets/lib/angular-sanitize.min.js') }}"></script>
     <script src="{{ asset('bundles/os2displayscreen/assets/lib/locale_da.js') }}"></script>
+    <script src="{{ asset('bundles/bibsdbsonderborgcalendar/assets/libs/angular-filter.min.js') }}"></script>
 
     <script src="{{ asset('bundles/os2displayscreen/app/fakeOffline.js') }}"></script>
     <script src="{{ asset('bundles/os2displayscreen/app/app.js') }}"></script>
@@ -35,6 +38,7 @@
     <script src="{{ asset('bundles/os2displayscreen/app/slide.js') }}"></script>
     <script src="{{ asset('bundles/os2displayscreen/app/socket.js') }}"></script>
     <script src="{{ asset('bundles/os2displayscreen/app/pull.js') }}"></script>
+    <script src="{{ asset('bundles/bibsdbsonderborgcalendar/apps/toolsModule/filters/groupedEvents.js') }}"></script>
   {% endif %}
 
   {# Initialize the relevant strategy #}
